_includes/footer.html

```diff
@@ -2,7 +2,7 @@
   <div class="padded-content">
     <div class="row align-items-center">
       <div class="col-12 col-md-3 footer-icon">
-        <img src="images/cfc-logo-icon.png" alt="Code for Chicago Logo Icon"/>
+        <img src="images/cfc_logo_short.svg" alt="Code for Chicago Logo Icon"/>
```

Comment:
Icon is looking a bit larger than it needs to be :)

<img width="1439" alt="Screen Shot 2021-02-10 at 8 59 37 PM" src="https://user-images.githubusercontent.com/9326234/107598321-045ec500-6be3-11eb-89c5-1b65f6f75cca.png">

Reply:
Ok, the size is smaller. I tried to go off of the design on figma to fix it accordingly. It really was comically large!